video_encoding/utils.py

```diff
@@ -0,0 +1,38 @@
+import os
+import tempfile
+
+
+def get_fieldfile_local_path(fieldfile):
```

Comment:
Would be great, if we could add a test for this. I know the current folder structure is a bit weird, but I would suggest adding `test_proj/video_encoding/test_utils.py` for now. I'll refactor the tests into a separate directory later.